nextflow.config

```diff
@@ -13,15 +13,32 @@ params {
     input                      = null
 
     // References
-    genome                     = null
+    genome                     = 'GRCh38'
     igenomes_base              = 's3://ngi-igenomes/igenomes'
     igenomes_ignore            = false
+    save_reference             = false
+    aligner                    = 'star'
```

Comment:
I can't find where the `params.aligner` is used.

Reply:
it is used in `subworkflows/local/prepare_genome.nf`